### PR TITLE
Fix some typo

### PR DESCRIPTION
**Pull request by 928234269**
Signed-off-by: Sakura <longfei.shang@daocloud.io>

**What this PR does / why we need it**:
This PR corrected grammar in docs/charts.md and fixes a simple typo in pkg/storage/storage.go, pkg/kube/client.go
**Special notes for your reviewer**:

**If applicable**:
- [x] this PR contains documentation

### CODE REVIEW

**Comment by 928234269** [comment]
@thomastaylor312 The signature of this pr is good, please review it here.

File: pkg/storage/storage.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func (s *Storage) Get(name string, version int32) (*rspb.Release, error) {
 
 // Create creates a new storage entry holding the release. An
 // error is returned if the storage driver failed to store the
-// release, or a release with identical an key already exists.
+// release, or a release with identical a key already exists.
```
Comment (thomastaylor312) [inline pkg/storage/storage.go]: I missed this the first time somehow. You can actually just remove the "a" to make it correct

Comment (928234269) [reply pkg/storage/storage.go]: as you wish.already fixed

Comment (928234269) [reply pkg/storage/storage.go]: > I missed this the first time somehow. You can actually just remove the "a" to make it correct

as you wish.already fixed

**Pull request closed without merging**